packages/build-utils/src/get-ignore-filter.ts

```diff
@@ -0,0 +1,81 @@
+import path from 'path';
```

Comment:
This entire file is 100% copy paste from [here](https://github.com/vercel/api/blob/31aba6fe60a5aa93990d36a7cc15d92c4766e659/scripts/build-container/src/utils/purpose-obtain.ts#L310). I'd like to make the Build Container load this file afterwards.